conn.go

```diff
@@ -263,11 +263,12 @@ func (c *Conn) handleMail(arg string) {
 			return
 		}
 	}
-	from := strings.Trim(fromArgs[0], "<> ")
+	from := strings.Trim(fromArgs[0], " ")
```

Comment:
No need for `strings.Trim` anymore. The string is guaranteed to be already trimmed since `fromArgs` is the result of `strings.Split`.

Reply:
Good point. Will test from := fromArgs[0]
tomorrow